packages/elements/src/components/toggle/toggle.tsx

```diff
@@ -79,11 +80,15 @@ export const ToggleRadio: ToggleRadioWrapped = forwardRef(
               name={name}
               value={value}
               type="radio"
+              disabled={disabled}
```

Comment:
Small point but you don't need to pass disabled here as it's already being passed by the ...rest operator

Reply:
just went to remove and I'm using the disabled prop elsewhere so unable to default to ...rest unless I change `htmlFor={!disabled ? id : undefined}` to `rest.disabled`?

Reply:
Ahh, cool